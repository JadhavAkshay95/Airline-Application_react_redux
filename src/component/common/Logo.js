import React from 'react'
import logo from '../../images/logo.jpg'

var style = {
  height: '50px',
  width: ' 50px',
  margin: '10px',
  position: 'relative',
  top: '-11px'
}

const Logo = () => {
  return (
    <div
      style={{
        position: 'absolute',
        left: '0'
      }}
    >
      <img style={style} src={logo} alt="tt" />
      <div style={{ display: 'inline-block' }}>
        <h1>AIRLINE</h1>
      </div>
    </div>
  )
}

export default Logo
