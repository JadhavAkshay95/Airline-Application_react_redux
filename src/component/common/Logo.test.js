import React from 'react'
import { shallow } from 'enzyme'
import Logo from './Logo'

describe('Render logo component', () => {
  it('Render logo ', () => {
    const wrapper = shallow(<Logo />)
    expect(wrapper).toMatchSnapshot()
  })
})
