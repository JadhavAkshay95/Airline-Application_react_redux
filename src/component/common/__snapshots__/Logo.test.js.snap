// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render logo component Render logo  1`] = `ShallowWrapper {}`;
